### ISSUE TITLE
Russian speech synthesis

### ISSUE THREAD

**Issue by sxdxfan** [NVIDIA/tacotron2#72]
Hi,

I'm trying to synthesise Russian speech and I'm not sure whether the network has learnt anything or not. The loss is decreasing, but the attention plots look all wrong.

<img width="1055" alt="screen shot 2018-08-17 at 18 35 17" src="https://user-images.githubusercontent.com/15383995/44275211-c5876880-a24c-11e8-9e1b-24b13a01f089.png">
<img width="622" alt="screen shot 2018-08-17 at 18 35 34" src="https://user-images.githubusercontent.com/15383995/44275219-ca4c1c80-a24c-11e8-8f51-7b0df5b9bb9c.png">
<img width="1055" alt="screen shot 2018-08-17 at 18 35 56" src="https://user-images.githubusercontent.com/15383995/44275223-ccae7680-a24c-11e8-8dbc-7b5058199280.png">

I've just modified the sample rate in hyperparameters to 44100 to match the audio files and all of the paths to text and audio. I've also replaced the symbols set to Russian and used only basic cleaners for text preprocessing. As for the data I have about 3 hours of speech recorded in a studio split into small 16 bit wavs.

Is my dataset too small or have I trained for a too small period of time? I've checked the Russian characters encoding, everything seems to be okay. I've also tried out the algorithms with LJSpeech dataset, which is 8 times bigger, it did train quite well.

**Comment by Evil-Spirit** [comment]
Hello. Do you have any update on this?

**Comment by sxdxfan** [comment]
Not really, we've decided to record more audio and see if it helps, I think we'll have an update in a week time.

**Comment by Rai220** [comment]
@sxdxfan what size of dataset do you use?

**Comment by sxdxfan** [comment]
> @sxdxfan what size of dataset do you use?
3 hour dataset of ~10 second clips. We've recorded more, but haven't tested yet.

**Comment by zuzana** [comment]
Hi @sxdxfan , do you have insight how many hours is necessary? Was 3 hours enough for you, or did you need more? I know that LJSpeech has 24hours. Thanks.

**Comment by Worldexe** [comment]
3hrs is not enough, afaik. You ideally should have 100hrs of well-done dataset (without noises and errors); 30+ may work.

**Comment by Evil-Spirit** [comment]
Why not take any of audio books? Some of them have good qualiity of sound/voice/speech.

**Comment by Worldexe** [comment]
Yeah, that will help.

Ideally, there should be intonation and overall speaking style consistent over all the records you have in the dataset. It may be an issue if you have the same speaker in those audiobooks, because small details, recording conditions will differ.

Also, there may be legal reasons against it.

**Comment by Evil-Spirit** [comment]
First of all, it can help to determine how much data you should take to get a good result. And can help in proving of concept, disclose any issues with dataset and point how you should record your dataset if you wanted to do it. Anyway, it can be cheaper if you pay for already recorded dataset instead of creating your own. And they are more natural!

**Comment by rafaelvalle** [comment]
MAILabs has a russian speech dataset. 20h is enough.

**Comment by rafaelvalle** [comment]
Closing due to inactivity.

**Comment by AlexanderKozhevin** [comment]
@rafaelvalle Sorry for stupid question. Can we train train network in any language (except asian ones)?

**Comment by asarsembayev** [comment]
> @rafaelvalle Sorry for stupid question. Can we train train network in any language (except asian ones)?

yes, you can. you should simply edit the `symbols.py`
